src/InitiateState.cpp

```diff
@@ -400,7 +415,7 @@ void* __fastcall do_game_update_new(void* thislol, int edx, int* a, int* b){
 		return do_game_update(thislol, a, b);
 	}
 
-	lua_State* L = (lua_State*)*((void**)thislol);
+	lua_State* L = reinterpret_cast<lua_State *>(thislol);
```

Comment:
Yeah you're right, good spot. It doesn't actually matter though since the variable's not used... I probably should just removed it. If it's actually a ** then why is the signature for a void *?